### PR TITLE
Specify bond type

### PR DESCRIPTION
**Pull request by calvix**
* add option to specify bond type to the mayu config
* add proper vlan template
* bond template update

### CODE REVIEW

File: pxemgr/config.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ type Network struct {
 	SubnetSize   string `yaml:"subnet_size"`
 	VlanId       string `yaml:"vlan_id"`
 	NetworkModel string `yaml:"network_model"`
+	BondMode     string
```
Comment (r7vme) [inline pxemgr/config.go]: so in config.yaml it will look like `bondmode: lacp` ?

Should we use `BondMode     string yaml:"bond_mode" `?

Comment (calvix) [inline pxemgr/config.go]: oh true, forget about that, thanks

**Comment by calvix** [comment]
@r7vme its kinda legacy thing :(

**Comment by r7vme** [comment]
@calvix LGTM

**Pull request merged**